frontend/package.json

```diff
@@ -53,8 +53,10 @@
     "typescript": "4.9.3"
   },
   "dependencies": {
+    "@11ty/eleventy-fetch": "3.0.0",
     "algoliasearch": "4.14.2",
     "alpinejs": "3.10.3",
+    "dotenv": "16.0.3",
```

Comment:
Remove these two packages as not used anymore.

Reply:
`eleventy-fetch`  and `dotenv` packages removed.